File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLTextBoxProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLTextBoxProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public String WaterMarkText;
        public String WaterMarkTextColor;
        public String WaterMarkTextFontString;
        public String TextColor;
        public String TextHeight;
        public String TextFontString;
        public String MaxChars;
        public String AllowedCharsRegEx;
        public String IsPassword;
        public String PasswordChar;
        public String HasBorder;
        public String BorderColor;
        public String BorderLineWidth;
        public String HasShadow;
        public String ShadowOffsetX;
        public String ShadowOffsetY;
        public String ShadowBlurValue;
        public String HasRoundedEdges;
        public String EdgeRadius;
        public String HasBgGradient;
        public String BgGradientStartColor;
        public String BgGradientEndColor;
        public String HasBgImage;
        public String BgImageUrl;
        public String HasAutoComplete;
        public List<Object> ListPossibles;
        public String DropDownPossiblesListIfThereIsInputText;
        public String LimitToListPossibles;
        public String ListPossiblesTextHeight;
        public String ListPossiblesTextFontString;
        public String CaretPosIndex;
        public String UserInputText;
        public String ShadowColor;
        public String ShowCaret;
        public String CaretColor;
        public Object Tag;
        public String CaretTime;
        public String TabStopIndex;

        CCLTextBoxProps()
        {
            ListPossibles = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLTextBoxProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLTextBoxProps
    {
        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public string WaterMarkText { get; set; }
        public string WaterMarkTextColor { get; set; }
        public string WaterMarkTextFontString { get; set; }
        public string TextColor { get; set; }
        public int TextHeight { get; set; }
        public string TextFontString { get; set; }
        public int MaxChars { get; set; }
        public string AllowedCharsRegEx { get; set; }
        public int IsPassword { get; set; }
        public string PasswordChar { get; set; }
        public int HasBorder { get; set; }
        public string BorderColor { get; set; }
        public int BorderLineWidth { get; set; }
        public int HasShadow { get; set; }
        public int ShadowOffsetX { get; set; }
        public int ShadowOffsetY { get; set; }
        public int ShadowBlurValue { get; set; }
        public int HasRoundedEdges { get; set; }
        public int EdgeRadius { get; set; }
        public int HasBgGradient { get; set; }
        public string BgGradientStartColor { get; set; }
        public string BgGradientEndColor { get; set; }
        public int HasBgImage { get; set; }
        public string BgImageUrl { get; set; }
        public int HasAutoComplete { get; set; }
        public List<string> ListPossibles { get; set; }
        public string DropDownPossiblesListIfThereIsInputText { get; set; }
        public int LimitToListPossibles { get; set; }
        public int ListPossiblesTextHeight { get; set; }
        public string ListPossiblesTextFontString { get; set; }
        public int CaretPosIndex { get; set; }
        public string UserInputText { get; set; }
        public string ShadowColor { get; set; }
        public int ShowCaret { get; set; }
        public string CaretColor { get; set; }
        public string CaretTime { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLTextBoxProps()
        {
            ListPossibles = new List<string>();
        }
    }
}